aziotctl/aziotctl-common/src/system_logs.rs

```diff
@@ -0,0 +1,17 @@
+use std::process::Command;
+
+#[allow(clippy::module_name_repetitions)]
+pub fn get_system_logs(processes: &[&str], mut additional_args: &[&str]) {
```

Comment:
`additional_args` should be `&[&OsStr]` in case the user gave non-utf-8 args (and it would be used with `Command::args_os` instead of `Command::args`).